src/main/resources/migration/sql/001_Basic_tables.sql

```diff
@@ -0,0 +1,68 @@
+CREATE SCHEMA messaging;
+CREATE TABLE IF NOT EXISTS messaging.person
+(
+    person_id uuid NOT NULL,
+    name character varying(50) NOT NULL,
+    date_of_birth timestamp with time zone,
+    person_info character varying(300),
+    CONSTRAINT person_table_pk PRIMARY KEY (person_id)
+);
+COMMENT ON TABLE messaging.person IS 'Данные пользователя';
+COMMENT ON COLUMN messaging.person.person_info IS 'Информация в вольном стиле от пользователя о себе';
+CREATE TABLE IF NOT EXISTS messaging.dialog
+(
+    message_id uuid NOT NULL,
+    dialog_id uuid NOT NULL,
+    writer_id uuid NOT NULL,
+    reader_id uuid NOT NULL,
+    create_time timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    text text,
+    CONSTRAINT dialog_table_pk PRIMARY KEY (message_id),
+    CONSTRAINT companion_id_fk FOREIGN KEY (reader_id)
+        REFERENCES messaging.person (person_id),
```

Comment:
@OlegKoshelev, usually it's not a good idea due to we can literally delete almost all db by deleting one row

Also, it's a good practice to use a soft delete

Reply:
@AlekseiLitkovetc 
Ok, let's imagine that we want to use soft deletion. In this case we will use field 'hidden' or something like that (or am I'm wrong?). 
But when we realy want to delete some info from our tables mb it's more comfortable use cascade delete (also it's save us from inappropriate information in reference tables with fk).

Reply:
> Ok, let's imagine that we want to use soft deletion. In this case we will use field 'hidden' or something like that (or am I'm wrong?).

Yes, usually a additional column is used. The most common is `deleted_at`.

> But when we realy want to delete some info from our tables mb it's more comfortable use cascade delete (also it's save us from inappropriate information in reference tables with fk)?

Yes, when we really need to permanently delete data it's very convenient of using cascade delition (and we can use it). But we should be extremly careful about it due to implicit deletion. One careless deletion and we will lost data forever (or we will go to backup in attemp to find a lost data).

So, from my practice, it's better to avoid cascade deliting option.

![image](https://user-images.githubusercontent.com/15446342/223211385-bf81bb3a-24a0-4c9a-9aab-b35f59965a78.png)